src/hooks/useLoadAccountLateData.ts

```diff
@@ -2,17 +2,21 @@ import { useCallback } from 'react';
 import { promiseUtils } from '../utils';
 import { prefetchAccountENSDomains } from './useAccountENSDomains';
 import useAccountSettings from './useAccountSettings';
+import useWallets from './useWallets';
 import logger from 'logger';
 
 export default function useLoadAccountLateData() {
   const { accountAddress } = useAccountSettings();
+  const { isReadOnlyWallet } = useWallets();
 
   const load = useCallback(async () => {
     logger.sentry('Load wallet account late data');
     return promiseUtils.PromiseAllWithFails([
-      prefetchAccountENSDomains({ accountAddress }),
+      ...(!isReadOnlyWallet
```

Comment:
wondering why don't we want to fetch this, is it because this is the list of ens names used in ens intro screen?